<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonorRegClass</class>
 <widget class="QMainWindow" name="DonorRegClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blood Bank Management System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RecipientReg.qrc">
    <normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</normaloff>:/RecipientReg/toppng.com-blood-donation-png-picture-blood-donation-logo-3209x3905.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="backimg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/RecipientReg/bloodd.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="RecipientReg.qrc">:/RecipientReg/bloodd.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idTB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mailTB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>106</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordTB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>166</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="maleRB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="femaleRB">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>220</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color:red</string>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Blood Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="idalertmsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="mailalertmsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordalertmsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="genderalertmsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameTB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ageTB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>106</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
    </widget>
    <widget class="QLabel" name="agealertmsg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw2TB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>166</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

QLineEdit:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="cpalertmsg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>Please make sure passwords match</string>
     </property>
    </widget>
    <widget class="QComboBox" name="bloodTypeTB">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCompoPox{
border:1px solid rgb(130, 130, 130) ;
border-radius:15px solid;
}

CompoPox:focus{
border:1px solid rgb(175, 175, 175) ;
border-radius:15px solid;
border-bottom:2px solid rgb(157, 0, 3);
}
</string>
     </property>
     <item>
      <property name="text">
       <string>A+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AB+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AB-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O-</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="namealertmsg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>please provide a valid value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>331</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px solid rgb(177, 0, 3);
border-radius:15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Date Of Lateset Donate</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateolatest">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="maximumDate">
      <date>
       <year>9999</year>
       <month>10</month>
       <day>30</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>d/M/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label_2</zorder>
    <zorder>idTB</zorder>
    <zorder>label_3</zorder>
    <zorder>mailTB</zorder>
    <zorder>label_5</zorder>
    <zorder>passwordTB</zorder>
    <zorder>label_8</zorder>
    <zorder>maleRB</zorder>
    <zorder>femaleRB</zorder>
    <zorder>label_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_9</zorder>
    <zorder>idalertmsg</zorder>
    <zorder>mailalertmsg</zorder>
    <zorder>passwordalertmsg</zorder>
    <zorder>genderalertmsg</zorder>
    <zorder>nameTB</zorder>
    <zorder>ageTB</zorder>
    <zorder>agealertmsg</zorder>
    <zorder>pw2TB</zorder>
    <zorder>cpalertmsg</zorder>
    <zorder>bloodTypeTB</zorder>
    <zorder>namealertmsg</zorder>
    <zorder>date</zorder>
    <zorder>dateolatest</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(177, 0, 3);
border-radius:25px;
color:white;
font: 700 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Arial&quot;;
background-color:rgb(240, 240, 240)</string>
    </property>
    <property name="text">
     <string>  Donor Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>521</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(57, 163, 255);
background-color:rgb(240, 240, 240);
border-radius:25px;
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RecipientReg.qrc"/>
 </resources>
 <connections/>
</ui>
